docs/react-testing-library/example-intro.md

```diff
@@ -134,7 +134,7 @@ export default function Fetch({url}) {
   return (
     <div>
       <button 
-        onClick={fetchGreeting} 
+        onClick={()=>fetchGreeting(url)} 
```

Comment:
What if instead, we leave this line as-is, and remove the `url` parameter from `fetchGreeting` above.

Reply:
Hi there @kentcdodds ,  I added the url b/c I saw in the example tests where the <Fetch url={url} />....so I was trying to reverse engineer the fetch.js component to match up this test example 
![image](https://user-images.githubusercontent.com/22228809/63354042-f64b2380-c320-11e9-8b89-fdd8dd0335b0.png)  Let me know what you think. Thanks

Reply:
Right, my suggestion will work with that as well :) What you have here is functionally equivalent to what I'm suggesting. The difference is that what's here right now is creating an extra function unnecessarily.

Reply:
Yes, I misunderstood...I see that we don't need to add url as a param to fetchGreeting because we have access to that prop inside the function already...got it...will update.  Thanks @kentcdodds , overlooked the obvious here :)